docs/applications.md

```diff
@@ -17,8 +17,18 @@ limitations under the License.
 
 # Fluo Applications
 
-Once you have Fluo installed and running on your cluster, you can now run Fluo applications which
-consist of clients and observers.
+Once you have Fluo installed and running on your cluster, you can run Fluo applications consisting
+of clients and observers.  A client is a user started process that uses the Fluo API.  An Observer
+is user provided code run by Fluo Workers in response to notifications.  Notifications are set by
+Fluo transactions executing in a client or observer.  Clients and observers rely on a running Fluo
+oracle service.  This documentations will shows how to :
+
+ * Create a Fluo client
+ * Create a Fluo observer
+ * Initialize a Fluo Application
+ * Start and stop Fluo Oracle and Worker processes
```

Comment:
Could be `Start and stop a Fluo application (which consists of oracle and worker processes)`